bluetest/integrationTestsBlue/integrationHelpersBlue.ts

```diff
@@ -0,0 +1,60 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import * as path from 'path';
+import { CSharpExtensionExports } from '../../src/csharpExtensionExports';
+
+export async function restartRoslynLanguageServer(): Promise<void> {
+    const csharpExtension = vscode.extensions.getExtension<CSharpExtensionExports>('ms-dotnettools.csharp');
+    if (!csharpExtension) {
+        throw new Error('Failed to find installation of ms-dotnettools.csharp');
+    }
+
+    if (!csharpExtension.isActive) {
+        await activateCSharpExtension();
```

Comment:
This won't necessarily restart the server.  If activation is complete and the runtime extension is installed then nothing will happen

Reply:
You probably want invoke the restart server or reload command explicitly